syshook_utils.c

```diff
@@ -0,0 +1,64 @@
+struct sys_hook {
```

Comment:
consider moving it to `syshook_utils.h`

Reply:
Correct me if i'm wrong, but this struct is indeed under syshook_utils.

Reply:
Yep, but inside a .c file

Reply:
moved to syshook_utils.h